gigantum/gigantum-client
# Set default server for cloud client in dev cluster

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
Small fix to properly set the default server when running the cloud configured client in the dev server.

**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
Should be able to launch a project in the dev cluster and it's able to import a project (because it's connecting to the right server)
